ports/stm32/boards/STM32H7B3I_DK/mpconfigboard.h

```diff
@@ -39,20 +39,20 @@
 // SMPS configuration
 #define MICROPY_HW_PWR_SMPS_CONFIG  (PWR_DIRECT_SMPS_SUPPLY)
 
-#if 0
+#if 1
```

Comment:
This could be `!MICROPY_HW_ENABLE_INTERNAL_FLASH_STORAGE` so that one can select between internal or external flash storage.